### ISSUE TITLE
Error when running `maloja info`

### ISSUE THREAD

**Issue by KucharczykL** [krateng/maloja#190]
```
Configuration Directory: /data
Data Directory: /data
Log Directory: /data/logs
Traceback (most recent call last):
  File "/usr/bin/maloja", line 8, in <module>
    sys.exit(main())
  File "/usr/lib/python3.9/site-packages/doreah/control.py", line 51, in wrapper
    result = func(*args,**kwargs)
  File "/usr/lib/python3.9/site-packages/maloja/__main__.py", line 178, in main
    return action(*args,**kwargs)
  File "/usr/lib/python3.9/site-packages/maloja/__main__.py", line 140, in print_info
    print(col['lightblue']("Network:                "),f"IPv{ip_address(conf.malojaconfig['host']).version}, Port {conf.malojaconfig['port']}")
  File "/usr/lib/python3.9/ipaddress.py", line 53, in ip_address
    raise ValueError(f'{address!r} does not appear to be an IPv4 or IPv6 address')
ValueError: '*' does not appear to be an IPv4 or IPv6 address
```

**Comment by bbsixzz** [comment]
I reported this here https://github.com/krateng/maloja/issues/186#issuecomment-1416932951

Try it and let us know how it goes.

**Comment by KucharczykL** [comment]
Thanks @bbsixzz! However, I think it should be fixed so it doesn't throw an exception OR be documented that it needs to be set first. That said, running it in Docker, I'm not sure what's the benefit of setting the IP manually.

**Comment by bbsixzz** [comment]
Sure, we're all awaiting feedback from @krateng

**Comment by krateng** [comment]
Hey, sorry for the late feedback, this was fixed in d8f53a5, so it will work in the next release.
